src/autocapture.js

```diff
@@ -35,7 +35,10 @@ var autocapture = {
         if (usefulElements.indexOf(tag_name) > -1) props['$el_text'] = getSafeText(elem)
 
         var classes = getClassName(elem)
-        if (classes.length > 0) props['classes'] = classes.split(' ')
+        if (classes.length > 0)
+            props['classes'] = classes.split(' ').filter(function (c) {
```

Comment:
Nit: We can use es6 `classes.split(' ').filter((className) => className.length > 0)`